litellm/tests/test_streaming.py

```diff
@@ -2214,15 +2214,15 @@ class ToolCalls(BaseModel):
 
 class Delta(BaseModel):
     role: str
-    content: Optional[str]
+    content: Optional[str] = None
```

Comment:
@msabramo any reason why the pydantic PR also involves setting these values to None? 

We've seen issues during streaming when setting default values like this

Reply:
Nah, no particular reason. I think those came from me running this:

```shell
bump-pydantic litellm
```

I just went and reverted those changes in bdb995f43.